### PR TITLE
[REVIEW] libgdf_cffi renamed to libcudf_cffi

### PR DESCRIPTION
**Pull request by pradghos**
This PR is addressing https://github.com/rapidsai/cudf/issues/1341

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by harrism** [comment]
We plan to replace CFFI with Cython in 0.7 release so this may be redundant.

**Comment by kkraus14** [comment]
ok to test

**Comment by pradghos** [comment]
I know `cffi` related changes will be removed probably once - issue#599 gets in. However, I am still marking as [REVIEW] - So that it can be merged as intermediate patch. Please let me know in case any comments. Thank you !

**Comment by pradghos** [comment]
Please keep me posted in case of any further review comments / suggestion . Thank you!

File: cpp/python/libcudf_cffi/tests/test_errorhandling.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    col = new_column()
+    gdf_dtype = get_dtype(dtype)
+
+    libcudf.gdf_column_view(col, ffi.NULL, ffi.NULL, 0, gdf_dtype)
+
+    # with pytest.raises(GDFError) as raises:
+    #     libcudf.gdf_add_generic(col, col, col)
+
+    # raises.match("CUDA ERROR.")
```
Comment (kkraus14) [inline cpp/python/libcudf_cffi/tests/test_errorhandling.py]: What's the goal of this test?

Comment (pradghos) [reply cpp/python/libcudf_cffi/tests/test_errorhandling.py]: @kkraus14 : This is existing test case only ! Probably after renaming the path from `libgdf_cffi` to `libcudf_cffi`, it is showing differently.  We can see there is another `test_errorhandling.py` being `marked` as deleted also.
 
So, there is nothing change as functionality is concern - Like all the places in code drop `libcudf_cffi` and `libcudf` is updated to pick up the intended module name. Thank you !

**Comment by harrism** [comment]
@kkraus14 Do we need to move this PR forward if we are just going to rip CFFI out in this release? If not, I recommend closing.

**Comment by kkraus14** [comment]
> @kkraus14 Do we need to move this PR forward if we are just going to rip CFFI out in this release? If not, I recommend closing.

Let's leave this in our back pocket in case that slips unexpectedly, so lets leave this open until then.

**Comment by harrism** [comment]
Looks like @kkraus14 has replaced nearly all CFFI with Cython and will complete the task in this release, so closing this PR. Thanks @pradghos and sorry for the waste of your effort.

To avoid that in the future, start a discussion in an issue first. I think you did file an issue, but needed to wait longer for a response before starting the work.

**Pull request closed without merging**